src/ServiceControlInstaller.PowerShell/Cmdlets/AuditInstances/NewServiceControlAuditInstance.cs

```diff
@@ -95,6 +95,11 @@ protected override void BeginProcessing()
             {
                 throw new Exception($"ConnectionString is mandatory for '{Transport}'");
             }
+            
+            if (TransportNames.IsDeprecated(Transport))
+            {
+                WriteWarning($"The transport '{Transport}' is deprecated.");
```

Comment:
This will come out as:

> The transport 'DEPRECATED: Azure Service Bus - Endpoint-oriented topology (Legacy)' is deprecated.

Given that all deprecated transports begin with the text 'DEPRECATED:' would it be worth just writing the name of the transport to the log and not bother with the rest of the message?

Also, is there any documentation about migration that we could direct a customer to?

Reply:
There will be a regular transport upgrade guide as usual on the docs site. Do you think we should log that as well?

Reply:
Improved it and pointed to the doco